### PR TITLE
Delete cypress

### PR DESCRIPTION
**Pull request by cesarvarela**

### CODE REVIEW

File: site/gatsby-site/cypress/support/e2e.js [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-
-// Alternatively you can use CommonJS syntax:
-// require('./commands')
-
-const ignoredErrors = [
-  // disable Realm error when origin is not whitelisted
-  'origin forbidden (status 403 Forbidden)',
-
-  // I have no Idea why this is happening, but it is.
-  'Transport not open',
```
Comment (kepae) [inline site/gatsby-site/cypress/support/e2e.js:29]: adios

File: site/gatsby-site/cypress/e2e/unit/functions/apiRequest.cy.js [hunk 2]
```diff
@@ -1,62 +0,0 @@
-const apiRequest = require('../../../../../realm/functions/apiRequest');
```
Comment (kepae) [inline site/gatsby-site/cypress/e2e/unit/functions/apiRequest.cy.js:1]: do we have a corresponding test for this in playwright? or is this just on it's way out in general?

**Review by kepae — COMMENTED** [review]
@clari182 do you see anything else before proceeding? (as you're updating the algolia test and probably stopping the migrations tests)

File: site/gatsby-site/cypress/e2e/unit/functions/apiRequest.cy.js (hunk 2 above)
Comment (cesarvarela) [reply site/gatsby-site/cypress/e2e/unit/functions/apiRequest.cy.js:1]: Deleted soon with the magic links stuff, and we are very very unlikely to modify any of those functions.

**Review by kepae — APPROVED** [review]

**Comment by clari182** [comment]
Looks good @kepae. AlgoliaUpdater is in another [pr](https://github.com/responsible-ai-collaborative/aiid/pull/3223) and the [PR](https://github.com/responsible-ai-collaborative/aiid/pull/3230) deletion of the migrations tests have already been merged.

**Comment by smcgregor** [comment]
🫡

**Pull request merged**